src/runner.rs

```diff
@@ -909,47 +928,12 @@ impl Log for RunnerLogger {
             return;
         }
 
-        let binding = record.args().to_string();
-        let msg_parts: Vec<&str> = binding.split_whitespace().collect();
-
-        if msg_parts.len() >= 4 {
-            println!(
-                "{} {} {} [{}] {}",
-                msg_parts[0],
-                msg_parts[1],
-                msg_parts[2],
-                record.target(),
-                msg_parts[3..].join(" ")
-            );
+        if let Some(log_file) = &self.log_file {
+            if let Ok(mut file) = log_file.write() {
```

Comment:
Acquiring the lock will only fail due to lock poisoning which we can treat as a logic error, so we can unwrap/expect here instead of a conditional.